### ISSUE TITLE
NPE from hyperlink detector

### ISSUE THREAD

**Issue by gavinking** [ceylon/ceylon-ide-eclipse#1391]
```
java.lang.NullPointerException
    at com.redhat.ceylon.model.typechecker.model.ModelUtil.getDirectMemberForBackend(ModelUtil.java:2526)
    at com.redhat.ceylon.model.typechecker.model.ModelUtil.getNativeHeader(ModelUtil.java:2575)
    at com.redhat.ceylon.eclipse.code.resolve.CeylonHyperlinkDetector.detectHyperlinks(CeylonHyperlinkDetector.java:138)
    at org.eclipse.jface.text.hyperlink.HyperlinkManager.findHyperlinks(HyperlinkManager.java:289)
    at org.eclipse.jface.text.hyperlink.HyperlinkManager.findHyperlinks(HyperlinkManager.java:261)
    at org.eclipse.jface.text.hyperlink.HyperlinkManager.mouseMove(HyperlinkManager.java:469)
```

**Comment by gavinking** [comment]
Looks like a bug un `CeylonHyperlinkDetector`. `containerToSearchHeaderIn` is `null` when I guess it should be the `Package`.

**Comment by davidfestal** [comment]
@gavinking : I know the reason of the error. But could you remind me on which declaration yàu got the exception ?

**Comment by gavinking** [comment]
No, I'm not sure, I just saw it in the log.

**Comment by davidfestal** [comment]
OK :-(

**Comment by gavinking** [comment]
Thanks.

On Thu, Jul 2, 2015 at 2:52 PM, David Festal <notifications@github.com>
wrote:

> Closed #1391 <https://github.com/ceylon/ceylon-ide-eclipse/issues/1391>
> via a1f798e
> <https://github.com/ceylon/ceylon-ide-eclipse/commit/a1f798ea803e34a9ea5f99a06aea64101a0b4b05>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ceylon/ceylon-ide-eclipse/issues/1391#event-346407788>
> .
>



-- 
Gavin King
gavin@ceylon-lang.org
http://profiles.google.com/gavin.king
http://ceylon-lang.org
http://hibernate.org
http://seamframework.org
